e2e/tests/omnibus-sepolia.toml/BFP_Market_Liquidations.e2e.js

```diff
@@ -0,0 +1,665 @@
+      (c) => c.collateralAddress === collateralAddress
+    );
+    log({ newDepositedsUSD });
+
+    assert.equal(parseFloat(ethers.utils.formatEther(newDepositedsUSD.available)), 25_000);
+  });
+
+  it('should open a short with fWETH collateral', async () => {
+    const collateralAddress = require('../../deployments/extras.json').weth_address;
+
+    const newDigest = await contractRead({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'getAccountDigest',
+      args: [accountId, marketId],
+    });
+    log({ newDigest });
+
+    const newDepositedWeth = newDigest.depositedCollaterals.find(
+      (c) => c.collateralAddress === collateralAddress
+    );
+    log({ newDepositedWeth });
+    const { now, blockTimestamp } = getTimes(provider);
+    const buffer = 1; // 1s
+    const diff = now - blockTimestamp;
+
+    if (diff < 0) {
+      await wait(Math.abs(diff) + buffer * 1000);
+    }
+    await commitBfpOrder({
+      wallet,
+      accountId,
+      marketId,
+      sizeDelta: -0.01,
+    });
+    await wait(1000);
+    await wait(5000);
+
+    const newPosition = await settleBfpOrder({ wallet, accountId, marketId });
+    assert.equal(newPosition.positionSize, -0.01);
+  });
+
+  it('should update market configuration minMarginUsd force liquidations', async () => {
+    let marketConfiguration = await contractRead({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'getMarketConfigurationById',
+      args: [marketId],
+    });
+    oldMinMarginUsd = marketConfiguration.minMarginUsd;
+    log({ marketConfiguration });
+
+    const marginDigest = await contractRead({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'getMarginDigest',
+      args: [accountId, marketId],
+    });
+    log({ marginDigest });
+    const collateralUsd = marginDigest.collateralUsd;
+    log({ collateralUsd });
+
+    // Sets minMarginUsd to be above the current collateral value
+    const updatedMarketConfiguration = {
+      ...marketConfiguration,
+      minMarginUsd: collateralUsd + 1,
+    };
+
+    const owner = await contractRead({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'owner',
+    });
+
+    await contractWrite({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'setMarketConfigurationById',
+      args: [
+        {
+          marketId,
+          ...updatedMarketConfiguration,
+        },
+      ],
+      impersonate: owner,
+    });
+  });
+
+  it('should flag an underwater fWETH backed position', async () => {
+    const { events } = await contractWrite({
+      wallet,
+      contract: 'BfpMarketProxy',
+      func: 'flagPosition',
+      args: [accountId, marketId],
+    });
+    let eventEmitted = false;
+    for (const event of events) {
+      if (event.event === 'PositionFlaggedLiquidation') {
+        eventEmitted = true;
+        console.assert(event.args.accountId.eq(accountId));
+        console.assert(event.args.marketId.eq(marketId));
```

Comment:
we dont have console.assert. Use `assert` instead

Reply:
Better to fill the array of necessary objects to check and do `assert.deepEqual` looking at full data here instead of list of small one-field asserts.
THis way in tests if there is an error we can see the nice diff between what's expected and what was actually there.

Reply:
I've updated tests using `deepEqual`